.travis.yml

```diff
@@ -10,10 +12,13 @@ notifications:
     on_failure: change
 before_install:
   - "echo 'gem: --no-ri --no-rdoc' > ~/.gemrc"
-before_script: 
+  - mysql -e 'CREATE DATABASE hackweek_test;'
```

Comment:
Agree that should work. I was following the recommendation from the Travis CI [docs for MySQL with ActiveRecord](https://docs.travis-ci.com/user/database-setup/#Using-MySQL-with-ActiveRecord). Will change to `bundle exec`